docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+# For local development run with a dummy provider
+version: '3'
+services:
+  aio-local:
+    build:
+      context: .
+      dockerfile: Dockerfile-aio
+    volumes:
+      - ".:/var/lib/vmpooler"
```

Comment:
We no longer need to map in this directory. Instead we should consider mapping in a configuration file to the default location, or specify one and map a file to that location, similarly to how we specify the configuration to marathon.